modules/ROOT/partials/configuration/exportpdf.adoc

```diff
@@ -1,7 +1,7 @@
 [[exportpdf-service-url]]
 == `exportpdf_service_url`
 
-The {pluginname} uses the {productname} Cloud Services HTML to PDF converter service to generate the `document-name.pdf` files.
+The {pluginname} uses the {productname} HTML to PDF converter service to generate the `document-name.pdf` files.
 
 NOTE: The {pluginname} feature is currently only available `on-premise` and requires the `exportpdf_service_url` option to be configured. https://www.tiny.cloud/contact/[Contact us] if you require this feature.
 
```

Comment:
@kemister85, how about changing the content to make it clearer that this is now on premises and needs a self-hosted service setup?

Proposal:

The Export to PDF feature is currently available exclusively for on-premise setups, requiring a self-hosted HTML to PDF converter service and the exportpdf_service_url option properly configured. If you need access to this feature, please contact us. Refer to our documentation for detailed instructions on deploying the TinyMCE Export to PDF service server-side component using Docker.

Link: https://www.tiny.cloud/docs/tinymce/latest/individual-export-to-pdf-on-premises

Tip:
Update the exportpdf_service_url configuration in your TinyMCE setup to point to your host. For instance, if your host is mycustomhost.com, the URL would become https://mycustomhost.com/v1/convert.

Reply:
Updated Proposal:

The Export to PDF feature is currently only available exclusively for `on-premise` setups, requiring a self-hosted HTML to PDF converter service and the exportpdf_service_url option to be properly configured. If you require access to this feature, please linkhttps://www.tiny.cloud/contact/[contact us]. Refer to our xref:individual-import-from-word-and-export-to-word-on-premises.adoc[documentation] for detailed instructions on deploying the {productname} {pluginname} service server-side component using Docker.

[TIP]
Update the `exportpdf_service_url` configuration in your {productname} setup to point to your host. For instance, if your host is `mycustomhost.com`, the URL would become `https://mycustomhost.com/v1/convert`.

Reply:
Various updates see: https://github.com/tinymce/tinymce-docs/pull/3397/commits/c0bf1ba6cf8677b46af1784b455b6794d64c6dce commit @ShiridiGandham